content/en/tracing/version_tracking/_index.md

```diff
@@ -0,0 +1,104 @@
+
+### Rolling Deploys
+
+Rolling deploys are a common way to do zero-downtime deployments.  The new version is replicated to all existing hosts or containers one-by-one, while traffic is directed to the other instances.
+
+Using Datadog, you can monitor your rolling deployments of new versions and well as quickly detect any error increases.
+
+{{< img src="tracing/version_tracking/rolling.png" alt="Versions on the Service Page"  style="width:100%;">}}
+
+### Blue/Green Deploys
+
+Blue/Green or other color combination deployments are a strategy to reduce downtime.  Depending on the particular configuration in your environments, this can either mean you are running two clusters of services that are both accepting traffic, or one is on standby, ready to be activated if there are problems with the other.
+
+In either case, setting and viewing the version tags for these services in Datadog allows the comparison requests and errors to detect if one of your clusters is having problems, whether that be an error rate higher than the other or than you SLOs, or if a cluster that is not supposed to be receiving traffic is.
+
+### Canary Deploys
+
+Canary deploys allow for a service to be deployed on a limited number of hosts or for a fraction of customers to test a new deployment with limited impact.
+
+Using Version tags within Datadog allows you to compare error rates, traces, and service behavior for the canary deployment.
+
+For example, you can see in the below image that a canary version was deployed, had a few errors, and was removed afterwards, with traces corresponding to that version available for investigation without any further impact to webservice integrity.
+
+{{< img src="tracing/version_tracking/Canary.png" alt="Versions on the Service Page"  style="width:100%;">}}
+
+### Shadow Deploys
+
+Using a Shadow Deployment strategy, a release candidate version is deployed to production to "shadow" the existing deployed service.   Incoming traffic is sent to both services, without impacting production users.
+
+Doing this allows testing a potential version against real production traffic.  Tagging these shadow deployments with a version tag within Datadog allows you to compare error rates, traces, and service behavior between the two versions to determine if the shadow version should be released.
+
+## Using Version Tags within Datadog
+
+Version tags can be used anywhere within Datadog, whether to filter a search view to a specific version, or compare versions.
+
+### The Service and Resource Page
+
+When version tags are present, the Service and Resource pages have te
+
+#### Service Page
```

Comment:
Too much heading-nesting. Let's just delete lines 55-57, and make line 59 and 75 an H3